mm2src/coins/utxo/slp.rs

```diff
@@ -1083,16 +1083,31 @@ impl MarketCoinOps for SlpToken {
 
     /// Receives raw transaction bytes in hexadecimal format as input and returns tx hash in hexadecimal format
     fn send_raw_tx(&self, tx: &str) -> Box<dyn Future<Item = String, Error = String> + Send> {
-        let coin = self.clone();
+        let _self = self.clone();
         let tx = tx.to_owned();
         let fut = async move {
             let bytes = hex::decode(tx).map_to_mm(|e| e).map_err(|e| format!("{:?}", e))?;
             let tx = deserialize(bytes.as_slice())
                 .map_to_mm(|e| e)
                 .map_err(|e| format!("{:?}", e))?;
-            let hash = coin.broadcast_tx(&tx).await.map_err(|e| format!("{:?}", e))?;
+            let hash = _self.broadcast_tx(&tx).await.map_err(|e| format!("{:?}", e))?;
             Ok(format!("{:?}", hash))
         };
+
+        Box::new(fut.boxed().compat())
+    }
+
+    fn send_raw_tx_bytes(&self, tx: &[u8]) -> Box<dyn Future<Item = String, Error = String> + Send> {
+        let _self = self.clone();
+        let bytes = tx.to_owned();
+        let fut = async move {
+            let tx = deserialize(bytes.as_slice())
+                .map_to_mm(|e| e)
+                .map_err(|e| format!("{:?}", e))?;
```

Comment:
I didn't notice before that this code can be optimized and simplified.
Since the future returns the `String` error, we can use `try_s!` macro here.

The `try_s!` macro adds a stack trace to the stringified error.
Currently, we're trying to replace `try_s!` by using `MmError` from the `common::mm_error` module. It performs exactly the same - tracks the stack trace of the error. But `send_raw_tx` has been added a long time ago, and it's overhead to refactor it within this PR.
`send_raw_tx_bytes` is very similar and generally calls the same common functions as `send_raw_tx`, so there is no need to refactor it as well.
In a nutshell, please consider using `try_s!` macro like:
```rust
let tx = try_s!(deserialize(bytes.as_slice()));
```